### PR TITLE
Test 2p responses

### PR DESCRIPTION
**Pull request by rcpeene**

### CODE REVIEW

**Comment by rcpeene** [comment]
This notebook is not yet done; I still need to clean it up and add some markdown. But I made this PR to open it up to review from @jeromelecoq

File: docs/first-order/test_2p_responses.ipynb [hunk 1]
```diff
@@ -0,0 +1,1092 @@
+{
```
Comment (jeromelecoq) [inline docs/first-order/test_2p_responses.ipynb:1]: <p>You should show the raw data to visualize the repeats</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (jeromelecoq) [inline docs/first-order/test_2p_responses.ipynb:1]: <p>Add labels to the axis</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (jeromelecoq) [inline docs/first-order/test_2p_responses.ipynb:1]: <p>Is it the max or mean that is used to measure the evoked response?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Comment (jeromelecoq) [inline docs/first-order/test_2p_responses.ipynb:1]: <p>Can you explain why 0 unit detected?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Comment (jeromelecoq) [inline docs/first-order/test_2p_responses.ipynb:1]: <p>Can you add a Venn diagram that compare the criteria to one another. I think there might be some python libraries to do that. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Comment (jeromelecoq) [inline docs/first-order/test_2p_responses.ipynb:1]: <p>Plot the distribution of your extracted metrics across all repeats. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

**Comment by jeromelecoq** [comment]
Approved. You have to resolve the conflict and you can merge

**Pull request merged**